src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenApplicant2UpdateApplication.java

```diff
@@ -28,7 +30,8 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
 
         configBuilder
             .event(CITIZEN_APPLICANT2_UPDATE)
-            .forStates(AwaitingApplicant2Response, AosDrafted, ConditionalOrderDrafted, ConditionalOrderPending, AwaitingClarification)
+            .forStates(ArrayUtils.addAll(AOS_STATES, AwaitingApplicant2Response, AosDrafted, ConditionalOrderDrafted,
```

Comment:
please use `EnumSet.of` as we have done in other events

Reply:
this doesnt work when theres multiple values. im not sure if Java has a spread operator like TS but it doesnt seem to work.